src/wlr_layout_ui/__init__.py

```diff
@@ -26,45 +51,38 @@ def main():
             print("")
             for p in profiles:
                 print(f" - {p}")
+        elif sys.argv[1] == "-m":
+            load()
+            current_uids = set(di.uid for di in displayInfo)
+            for key in sorted(profiles.keys()):
+                prof_uids = {p["uid"] for p in profiles[key]}
+                # check that the two sets have the same elements
+                if len(prof_uids & current_uids) == len(current_uids):
+                    print(f"Matched profile {key}. Applying it...")
+                    apply_profile(profiles[key])
+                    sys.exit(0)
+            print(f"No profile found: {sys.argv[1]}")
+            sys.exit(1)
+
         elif sys.argv[1][0] == "-":
             load()
             print(
                 """With no options, launches the GUI
 Options:
              -l : list profiles
+             -m : find a profile that matches the current displays layout, and apply it. 
```

Comment:
matches the current displays layout => matches the currently plugged displays set or something like this (it doesn't try to match the "layout", is it?)